builder.json

```diff
@@ -3,6 +3,19 @@
     "packages": [
         "ruby"
     ],
+    "targets": {
+        "linux": {
+            "variables": {
+                "openssl_include": "/opt/openssl/include",
+                "openssl_lib": "/opt/openssl/lib"
```

Comment:
ohhh yeah this weird crap
we need a gem/rake idiomatic way for the user to pass info all the way down to cmake
feel free to check this in as-is, or revert this file entirely
we can fight linux together tomorrow

Reply:
Yeah - I'll remove this for now.  Windows is working with minimal weirdness. Linux has more issues than just this...